common/mailUtils.js

```diff
@@ -0,0 +1,26 @@
+import {MailSlurp} from "mailslurp-client";
+
+let apiKey
+let mailslurp
+let inbox
+
+async function startService() {
+    apiKey = process.env.MAILSLURP_API_KEY
+    mailslurp = new MailSlurp({apiKey})
+    inbox = await mailslurp.inboxController.getInbox({inboxId: process.env.INBOX_ID})
+}
+
+export const getEmailAddress = async () => {
+    if (!apiKey && !mailslurp && !inbox) await startService()
+    return inbox.emailAddress
+}
+
+export const getNextMail = async () =>{
+    if (!apiKey && !mailslurp && !inbox) await startService()
```

Comment:
Pierdoła, ale jak coś będziesz zmieniać, to ten warunek wyrzuciłbym do funkcji `isServiceStarted` - DRY - albo w ogóle cały warunek i akcję do `startServiceIfNeeded`